met-web/src/components/engagement/form/EngagementFormTabs/UserManagement/AddTeamMemberModal.tsx

```diff
@@ -84,11 +84,11 @@ export const AddTeamMemberModal = () => {
         }, 1000),
     ).current;
 
-    const setErrors = (error: AxiosError) => {
+    const setErrors = (error: AxiosError<{ message?: string }>) => {
```

Comment:
I assume these type hints are required by the new Axios version?

Reply:
Yes the the Dependabot PR failed due to this.